app/src/main/kotlin/com/metrolist/music/ui/screens/settings/ThemeScreen.kt

```
@@ -179,6 +186,66 @@ fun ThemeScreen(
             }
         }
     )
+
+    var showCustomColorDialog by remember { mutableStateOf(false) }
+    if (showCustomColorDialog) {
+        var hexValue by remember { 
+            val currentHex = String.format("#%06X", (0xFFFFFF and selectedThemeColorInt))
```

detekt (1.23.8)

[warning] String.format("#%06X", (0xFFFFFF and selectedThemeColorInt)) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)